### ISSUE TITLE
Detect new indexes and other issues

### ISSUE THREAD

**Issue by klinux** [DarioBalinzo/kafka-connect-elasticsearch-source#53]
Hi,

First, thank you for this amazing connect plugin.

I have a lot of indexes, named by day, like **idx-2021.08.30**, the first problem that I disovered here in my environment, it's look like the elasticsearch connect freezes in some index, and the connect enter in type of loop:

```bash
cp-kafka-connect-connect-es-74df8cf9d6-92whw connect-es-server [2021-08-31 20:44:45,664] INFO index idx-2021.08.31 total messages: 295939  (com.github.dariobalinzo.task.ElasticSourceTask)
cp-kafka-connect-connect-es-74df8cf9d6-92whw connect-es-server [2021-08-31 20:44:56,592] INFO index idx-2021.08.31 total messages: 295939  (com.github.dariobalinzo.task.ElasticSourceTask)
cp-kafka-connect-connect-es-74df8cf9d6-92whw connect-es-server [2021-08-31 20:44:58,016] INFO index idx-2021.08.31 total messages: 295939  (com.github.dariobalinzo.task.ElasticSourceTask)
```
And this line repeat eternally.

Other problem is with new indexes, new indexes are created every day, but only after restart the connect task, connect can find the new index. And my indexes has only @timestamp as incremental field, but always has less document in topics comparing with index.

I appreciate so much if you can help me.

Thank you,

**Comment by DarioBalinzo** [comment]
Regarding the first problem, the use of the timestamp with many duplicates may produce these issue, since the incrementing field should be strictly incremental. However to fix this you can use the `incrementing.secondary.field.name` config, here you can find the documentation: https://github.com/DarioBalinzo/kafka-connect-elasticsearch-source/tree/1.4.2

For the new indexes discovering, this is an interesting feature, I will add that as a new task in the backlog.

**Comment by ilosamart** [comment]
Hello!!! Thanks for this great work!

I'm currently in need for this feature `indexes discovering` and have implemented it. Since I'm not a Java developer, I would pretty much appreciate if you guys could review my proposed PR.

**Comment by DarioBalinzo** [comment]
Hi @ilosamart,
thank for your contribution! I will be happy to review it soon. 
First of all I will update the master branch with the latest release and then I will check it in the next days.
